hazelcast/src/main/java/com/hazelcast/config/MemberDomConfigProcessor.java

```diff
@@ -1664,14 +1664,14 @@ private NearCacheConfig handleNearCacheConfig(Node node) {
             String value = getTextContent(child).trim();
             if ("max-size".equals(nodeName)) {
                 nearCacheConfig.setMaxSize(Integer.parseInt(value));
-                LOGGER.warning("The element <max-size/> for <near-cache/> is deprecated, please use <eviction/> instead!");
+                LOGGER.warning("The element <max-size> for <near-cache> is deprecated, please use <eviction> instead!");
```

Comment:
Are these intentional?

Reply:
Definitely not intentional. Should be fixed in the latest commit.